src/functions.js

```diff
@@ -51,7 +51,7 @@ defineFunction([
     return {
         type: "text",
         body: ordargument(body),
-        style: textFunctionStyles[context.funcName],
+        font: textFunctionStyles[context.funcName],
```

Comment:
Since we're calling these fonts now, should probably rename this to `textFunctionFonts`.